modules/lib/src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -2736,10 +2742,13 @@ export class ContentWizardPanel
     }
 
     private updateMissingOrStoppedApplications() {
+        const before = this.missingOrStoppedAppKeys?.length ?? 0;
+
         this.fetchMissingOrStoppedAppKeys().then((missingApps: ApplicationKey[]) => {
             this.missingOrStoppedAppKeys = missingApps;
+            const after = missingApps.length;
 
-            if (missingApps.length > 0) {
+            if (before !== after) { // can also sort arrays and check array equality
```

Comment:
This is too philosophical. Can we rename this to `currentAppCount` and `fetchedAppCount`?